tests/generation/test_generate_missing.py

```diff
@@ -39,13 +39,13 @@ def test_missing_steps():
 
     result.stdout.fnmatch_lines(
         [
-            'Step "I have a custom bar" is not defined in the scenario '
+            'Step "Given I have a custom bar" is not defined in the scenario '
```

Comment:
yep, had that in mind, just Given step "I have" is not english at all
Step Given "I have is probably better